OpenRA.Game/Game.cs

```diff
@@ -594,55 +597,57 @@ static void InnerLogicTick(OrderManager orderManager)
 				Cursor.Tick();
 			}
 
-			var worldTimestep = world == null ? Timestep : world.IsLoadingGameSave ? 1 : world.Timestep;
+			var worldTimestep = orderManager.SuggestedTimestep;
+
 			var worldTickDelta = tick - orderManager.LastTickTime;
```

Comment:
This appears to be the only place that uses `orderManager.LastTickTime` now, and this isn't updated reliably (only when `orderManager.TryTick` fails *and* `NetFrameNumber == 0`) so is basically a no-op in the new net code.

It looks like it should be possible to rewrite this using `orderManager.RealTickTime` &rarr; `TickStartRuntime` for both old and new netcodes, which would make this a lot clearer and hopefully less buggy (I imagine that the frame limiter bugs is also related).